demo/site/src/documents/pages/blocks/TeaserBlock.tsx

```diff
@@ -10,7 +10,7 @@ const TeaserBlock = withPreview(
         return (
             <Root>
                 <HeadlineBlock data={headline} />
-                <DamImageBlock data={image} />
+                <DamImageBlock data={image} aspectRatio="1x1" />
```

Comment:
Why is it `1x1` here? Seems random to me

Reply:
Yes, this is random. What would you prefer? `inherit` or `1x1`?

Reply:
Actually it doesn't really matter since it's just an example. I missed that this is in Demo.